README.md

```diff
@@ -109,6 +109,12 @@ $CONFIG = array (
     // Default group to add users to (optional, defaults to nothing)
     'oidc_login_default_group' => 'oidc',
 
+    // Authorize only the configured group to access Nextcloud. In case the user
+    // is not assigned to this group (read from oidc_login_attributes) the login
+    // will not be allowed for this user. When the user is not authorized, the user
+    // will neither be created nor its data updated.
+    'oidc_login_authorized_group' => 'admin',
```

Comment:
The example is a single group, but your code supports array of groups. I suggest either:
- make the example configuration an array
- support both single values and arrays

Reply:
Acutally the my new check does not support an array of groups (LoginController.php#213 and below). I just moved the original code a few lines higher to ensure that no user/group is created before my new check applies. This is why it is just required to check, that a single authorized group (as configured) is among the groups retrieved from the OIDC provider. So, I only verify if the configured group (not an array) is among those groups.

Should I really support multiple groups and allow configuration for an array and a single group? I thought it is enough to restrict it to just a single group, as you can put all users accessing Nextcloud that should be authorized into this new group.

Reply:
@azmeuk Please a short response how we should continue. Thanks!

Reply:
I think you are right. Let's start with support for a single group, and if someone someday needs support for multiple group, we can think again.